src/pal/pal_bsd.h

```diff
@@ -24,6 +24,8 @@ namespace snmalloc
      */
     static constexpr uint64_t pal_features = PALPOSIX<OS>::pal_features;
 
+    static constexpr size_t OS_PAGE_SIZE = 0x1000;
```

Comment:
It's probably sensible to put this in pal_posix (if it's useful in a superclass at all).